cirq-core/cirq/value/prng_test.py

```diff
@@ -0,0 +1,48 @@
+# Copyright 2024 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import List, Union
+import numpy as np
+import cirq
+
+
+class TestPrng(cirq.value.CustomPRNG):
```

Comment:
In general it is better to avoid the *Test* prefix and name local types as ExamplePrng / ExampleSomething.
[pytest](https://docs.pytest.org/en/7.1.x/getting-started.html#group-multiple-tests-in-a-class) uses Test-classes for defining and grouping the actual tests so on a first sight it is non-obvious what this class is for.

That said I would still prefer to take the CustomPRNG and associated testing out.